handsontable/src/helpers/dom/__tests__/element.spec.js

```diff
@@ -22,4 +22,27 @@ describe('DOM helpers', () => {
       document.body.removeChild(element);
     });
   });
+
+  describe('offset', () => {
+    it('should return correct offset for elements inside a foreign object', () => {
+      const wrapper = document.createElement('div');
+      wrapper.innerHTML = /* html */`
+        <svg xmlns="http://www.w3.org/2000/svg">
+          <foreignObject width="50" height="50">
+            <div xmlns="http://www.w3.org/1999/xhtml">test</div>
+          </foreignObject>
+        </svg>
+      `;
+
+      document.body.appendChild(wrapper);
+
+      const element = wrapper.querySelector('div');
+      const elementOffset = offset(element);
```

Comment:
I forgot to mention that in these tests, the DOM helpers are exposed in `Handsontable.dom` namespace. Should be:
```suggestion
      const elementOffset = Handsontable.dom.offset(element);
```

Reply:
Makes sense! Thanks.